diff --git a/node_modules/@stripe/stripe-react-native/android/build.gradle b/node_modules/@stripe/stripe-react-native/android/build.gradle
index c40c1a3..2bb12df 100644
--- a/node_modules/@stripe/stripe-react-native/android/build.gradle
+++ b/node_modules/@stripe/stripe-react-native/android/build.gradle
@@ -75,6 +75,10 @@ if (isNewArchitectureEnabled()) {
 }
 
 android {
+    composeOptions {
+        kotlinCompilerExtensionVersion "1.5.14"
+    }
+
   namespace "com.reactnativestripesdk"
   buildFeatures {
     buildConfig true
